### DISCUSSION TITLE
How to keep files on cloud and download it on demand?

### DISCUSSION THREAD

**Discussion by lamkr** [abraunegg/onedrive#2413] [Q&A] [answered]
In the OneDrive client for Windows it is possible to keep the files only in the cloud and they are downloaded on demand. How can I apply the same configuration on the client for Linux, to free space locally? 
Thank you very much.

**Comment by abraunegg** [comment, marked as answer]
@lamkr 
Please refer to the documentation: https://github.com/abraunegg/onedrive#readme

![image](https://github.com/abraunegg/onedrive/assets/4956234/c8cd280c-5323-4c4a-8fdb-58b9bec9dd49)

To use the OneDrive files 'on-demand' capability in Linux, please investigate the following GitHub project: https://github.com/jstaf/onedriver

**Reply by lamkr** [reply]
Oh, sorry! 
Thank you very much!
